discovery_engine/test/repository/hive_active_document_repo_test.dart

```diff
@@ -32,17 +32,24 @@ import '../logging.dart' show setupLogging;
 Future<void> main() async {
   setupLogging();
 
-  final box = await Hive.openBox<ActiveDocumentData>(
-    activeDocumentDataBox,
-    bytes: Uint8List(0),
-  );
-  final repo = HiveActiveDocumentDataRepository();
-
   group('ActiveDocumentDataRepository', () {
+    late Box<ActiveDocumentData> box;
+    late HiveActiveDocumentDataRepository repo;
     final data = ActiveDocumentData(Uint8List(0));
     final id1 = DocumentId();
     final id2 = DocumentId();
 
+    setUpAll(() async {
+      box = await Hive.openBox<ActiveDocumentData>(
+        activeDocumentDataBox,
+        bytes: Uint8List(0),
```

Comment:
`bytes` will be taken in consideration only the first time. This can be a bit misleading.

Reply:
My bad, it run only once.